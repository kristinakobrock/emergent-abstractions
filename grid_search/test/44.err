  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 25.08it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 25.03it/s] 18%|################7                                                                            | 9/50 [00:00<00:01, 24.75it/s] 24%|######################                                                                      | 12/50 [00:00<00:01, 23.69it/s] 30%|###########################6                                                                | 15/50 [00:00<00:01, 21.86it/s] 36%|#################################1                                                          | 18/50 [00:00<00:01, 23.31it/s] 42%|######################################6                                                     | 21/50 [00:00<00:01, 23.96it/s] 50%|##############################################                                              | 25/50 [00:01<00:00, 25.79it/s] 56%|###################################################5                                        | 28/50 [00:01<00:00, 25.60it/s] 62%|#########################################################                                   | 31/50 [00:01<00:00, 26.75it/s] 68%|##############################################################5                             | 34/50 [00:01<00:00, 26.27it/s] 74%|####################################################################                        | 37/50 [00:01<00:00, 27.28it/s] 80%|#########################################################################6                  | 40/50 [00:01<00:00, 27.96it/s] 86%|###############################################################################1            | 43/50 [00:01<00:00, 27.07it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 27.77it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 27.02it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.87it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 20.00it/s] 38%|###################################7                                                         | 5/13 [00:00<00:00, 23.43it/s] 62%|#########################################################2                                   | 8/13 [00:00<00:00, 26.03it/s] 85%|#############################################################################8              | 11/13 [00:00<00:00, 24.39it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.10it/s]
