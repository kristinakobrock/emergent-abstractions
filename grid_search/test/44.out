# ["--attributes=3", "--values=16", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6811423897743225, "acc": 0.6034983396530151, "length": 3.5267391204833984, "mode": "train", "epoch": 1}
{"loss": 0.682519793510437, "acc": 0.56640625, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6645743250846863, "acc": 0.6226117014884949, "length": 3.4210760593414307, "mode": "train", "epoch": 2}
{"loss": 0.6693918108940125, "acc": 0.619921863079071, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6602940559387207, "acc": 0.6269228458404541, "length": 3.386258125305176, "mode": "train", "epoch": 3}
{"loss": 0.6775796413421631, "acc": 0.5976563096046448, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.6596056222915649, "acc": 0.6268680095672607, "length": 3.3709936141967773, "mode": "train", "epoch": 4}
{"loss": 0.6709612011909485, "acc": 0.6169920563697815, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6580187678337097, "acc": 0.6254770159721375, "length": 3.2889585494995117, "mode": "train", "epoch": 5}
{"loss": 0.6674330234527588, "acc": 0.6214645504951477, "length": 1.03515625, "mode": "test", "epoch": 5}
{"loss": 0.6563968658447266, "acc": 0.6279175877571106, "length": 3.203202962875366, "mode": "train", "epoch": 6}
{"loss": 0.6539005041122437, "acc": 0.6259003281593323, "length": 2.66015625, "mode": "test", "epoch": 6}
{"loss": 0.6528997421264648, "acc": 0.6280768513679504, "length": 3.251085042953491, "mode": "train", "epoch": 7}
{"loss": 0.6500115394592285, "acc": 0.6288878917694092, "length": 3.71875, "mode": "test", "epoch": 7}
{"loss": 0.6441507935523987, "acc": 0.6306318044662476, "length": 3.3854289054870605, "mode": "train", "epoch": 8}
{"loss": 0.642772912979126, "acc": 0.6339783072471619, "length": 3.93359375, "mode": "test", "epoch": 8}
{"loss": 0.6270609498023987, "acc": 0.6489521861076355, "length": 3.630258798599243, "mode": "train", "epoch": 9}
{"loss": 0.6267255544662476, "acc": 0.6593749523162842, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.6092634797096252, "acc": 0.6735002994537354, "length": 3.8095133304595947, "mode": "train", "epoch": 10}
{"loss": 0.6222044825553894, "acc": 0.6703368425369263, "length": 3.984375, "mode": "test", "epoch": 10}
{"loss": 0.5964390635490417, "acc": 0.6845332980155945, "length": 3.6797780990600586, "mode": "train", "epoch": 11}
{"loss": 0.6222274899482727, "acc": 0.6728195548057556, "length": 3.98046875, "mode": "test", "epoch": 11}
{"loss": 0.5833430886268616, "acc": 0.6968045234680176, "length": 3.762686252593994, "mode": "train", "epoch": 12}
{"loss": 0.6079457998275757, "acc": 0.6904296875, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.5683816075325012, "acc": 0.7184842824935913, "length": 3.8452706336975098, "mode": "train", "epoch": 13}
{"loss": 0.5835822820663452, "acc": 0.7039061784744263, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.5511211156845093, "acc": 0.7290725708007812, "length": 3.913243055343628, "mode": "train", "epoch": 14}
{"loss": 0.5936865210533142, "acc": 0.7013671398162842, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.5381850004196167, "acc": 0.7400276064872742, "length": 3.942549228668213, "mode": "train", "epoch": 15}
{"loss": 0.5791621208190918, "acc": 0.7091797590255737, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.525593638420105, "acc": 0.7451691031455994, "length": 3.95869517326355, "mode": "train", "epoch": 16}
{"loss": 0.5728415846824646, "acc": 0.7259765267372131, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.5136947631835938, "acc": 0.7546218633651733, "length": 3.970968246459961, "mode": "train", "epoch": 17}
{"loss": 0.5762391686439514, "acc": 0.7230468392372131, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.5062268972396851, "acc": 0.7611826062202454, "length": 3.978282928466797, "mode": "train", "epoch": 18}
{"loss": 0.5558159351348877, "acc": 0.7316405773162842, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.4961030185222626, "acc": 0.764085590839386, "length": 3.9837498664855957, "mode": "train", "epoch": 19}
{"loss": 0.5538562536239624, "acc": 0.7359375357627869, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.48159259557724, "acc": 0.7759972214698792, "length": 3.9867780208587646, "mode": "train", "epoch": 20}
{"loss": 0.5383304357528687, "acc": 0.7384766936302185, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.47558507323265076, "acc": 0.7769125699996948, "length": 3.9896762371063232, "mode": "train", "epoch": 21}
{"loss": 0.5287229418754578, "acc": 0.7468750476837158, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.46884390711784363, "acc": 0.779451847076416, "length": 3.9910271167755127, "mode": "train", "epoch": 22}
{"loss": 0.5212299823760986, "acc": 0.7484374642372131, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.4567537307739258, "acc": 0.7903319597244263, "length": 3.994338274002075, "mode": "train", "epoch": 23}
{"loss": 0.5116305351257324, "acc": 0.7501951456069946, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.4526079297065735, "acc": 0.7924211025238037, "length": 3.996062994003296, "mode": "train", "epoch": 24}
{"loss": 0.5083670616149902, "acc": 0.7574219107627869, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.4435897171497345, "acc": 0.7958996295928955, "length": 3.9965169429779053, "mode": "train", "epoch": 25}
{"loss": 0.4882814884185791, "acc": 0.7718749642372131, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.4324827492237091, "acc": 0.8050441145896912, "length": 3.9954185485839844, "mode": "train", "epoch": 26}
{"loss": 0.47958993911743164, "acc": 0.7742187976837158, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.4309282600879669, "acc": 0.8052427172660828, "length": 3.9966418743133545, "mode": "train", "epoch": 27}
{"loss": 0.48546141386032104, "acc": 0.7755858898162842, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.4195350110530853, "acc": 0.8077888488769531, "length": 3.9987032413482666, "mode": "train", "epoch": 28}
{"loss": 0.4651268720626831, "acc": 0.7914062142372131, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.41119271516799927, "acc": 0.8174259066581726, "length": 3.9982054233551025, "mode": "train", "epoch": 29}
{"loss": 0.4591728448867798, "acc": 0.7949218153953552, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.40721395611763, "acc": 0.8161945343017578, "length": 3.9993174076080322, "mode": "train", "epoch": 30}
{"loss": 0.46539533138275146, "acc": 0.7910155653953552, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3996332585811615, "acc": 0.8208965063095093, "length": 3.9969160556793213, "mode": "train", "epoch": 31}
{"loss": 0.4780787229537964, "acc": 0.7880859971046448, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.39937788248062134, "acc": 0.8237892389297485, "length": 3.998218059539795, "mode": "train", "epoch": 32}
{"loss": 0.46630504727363586, "acc": 0.7935547232627869, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3875603973865509, "acc": 0.8266991376876831, "length": 3.9976232051849365, "mode": "train", "epoch": 33}
{"loss": 0.4523143768310547, "acc": 0.7994140982627869, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3825930058956146, "acc": 0.8309784531593323, "length": 3.998777151107788, "mode": "train", "epoch": 34}
{"loss": 0.44326335191726685, "acc": 0.8095702528953552, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.3756431043148041, "acc": 0.8358718156814575, "length": 3.9998326301574707, "mode": "train", "epoch": 35}
{"loss": 0.4414924383163452, "acc": 0.8052734732627869, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.3696185052394867, "acc": 0.8364688754081726, "length": 3.9998841285705566, "mode": "train", "epoch": 36}
{"loss": 0.43582695722579956, "acc": 0.8126954436302185, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.362722247838974, "acc": 0.8399736285209656, "length": 3.9993772506713867, "mode": "train", "epoch": 37}
{"loss": 0.43737101554870605, "acc": 0.8124999403953552, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.36129099130630493, "acc": 0.8414162397384644, "length": 3.9999260902404785, "mode": "train", "epoch": 38}
{"loss": 0.4430367946624756, "acc": 0.8095703125, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3631442189216614, "acc": 0.8425695300102234, "length": 3.9988250732421875, "mode": "train", "epoch": 39}
{"loss": 0.4304429292678833, "acc": 0.8128905892372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.36218902468681335, "acc": 0.8382582664489746, "length": 3.999957799911499, "mode": "train", "epoch": 40}
{"loss": 0.4282662868499756, "acc": 0.8236327767372131, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.35268884897232056, "acc": 0.846778929233551, "length": 3.999891519546509, "mode": "train", "epoch": 41}
{"loss": 0.4173036813735962, "acc": 0.818359375, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.35274118185043335, "acc": 0.8483874201774597, "length": 3.999917984008789, "mode": "train", "epoch": 42}
{"loss": 0.4369346797466278, "acc": 0.8158202171325684, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.3499307930469513, "acc": 0.8488050103187561, "length": 3.999875783920288, "mode": "train", "epoch": 43}
{"loss": 0.42409318685531616, "acc": 0.8160156607627869, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.34906262159347534, "acc": 0.8489863872528076, "length": 3.9999871253967285, "mode": "train", "epoch": 44}
{"loss": 0.4141899049282074, "acc": 0.8238281607627869, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.3504398763179779, "acc": 0.848449170589447, "length": 3.9999642372131348, "mode": "train", "epoch": 45}
{"loss": 0.4327709674835205, "acc": 0.8193359375, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.3436222970485687, "acc": 0.8532667756080627, "length": 3.9991188049316406, "mode": "train", "epoch": 46}
{"loss": 0.4195699691772461, "acc": 0.8222656846046448, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.3433242440223694, "acc": 0.8535158634185791, "length": 3.999994993209839, "mode": "train", "epoch": 47}
{"loss": 0.41476041078567505, "acc": 0.8292970061302185, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.3423920273780823, "acc": 0.8557804226875305, "length": 3.9999935626983643, "mode": "train", "epoch": 48}
{"loss": 0.40678977966308594, "acc": 0.8259764909744263, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.33291223645210266, "acc": 0.8617401719093323, "length": 3.9999969005584717, "mode": "train", "epoch": 49}
{"loss": 0.4141218364238739, "acc": 0.8275390267372131, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.3323362171649933, "acc": 0.8578068614006042, "length": 3.9999985694885254, "mode": "train", "epoch": 50}
{"loss": 0.42090904712677, "acc": 0.8230467438697815, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.3311019539833069, "acc": 0.8592965602874756, "length": 3.9999988079071045, "mode": "train", "epoch": 51}
{"loss": 0.40923237800598145, "acc": 0.8332031965255737, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.3288097083568573, "acc": 0.8599521517753601, "length": 3.999994993209839, "mode": "train", "epoch": 52}
{"loss": 0.40349888801574707, "acc": 0.8343749642372131, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.3311755359172821, "acc": 0.8582242131233215, "length": 3.9999985694885254, "mode": "train", "epoch": 53}
{"loss": 0.4135836362838745, "acc": 0.8318358659744263, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.3293999135494232, "acc": 0.8609851002693176, "length": 3.9993021488189697, "mode": "train", "epoch": 54}
{"loss": 0.40976446866989136, "acc": 0.8302735686302185, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.3199974000453949, "acc": 0.8658509254455566, "length": 3.9999797344207764, "mode": "train", "epoch": 55}
{"loss": 0.41252759099006653, "acc": 0.8316407203674316, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.3225259780883789, "acc": 0.8644812703132629, "length": 3.9999945163726807, "mode": "train", "epoch": 56}
{"loss": 0.4091658890247345, "acc": 0.8341797590255737, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.31601381301879883, "acc": 0.8689005374908447, "length": 3.999128818511963, "mode": "train", "epoch": 57}
{"loss": 0.41299375891685486, "acc": 0.8294922113418579, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.3141867518424988, "acc": 0.8668652772903442, "length": 3.9999990463256836, "mode": "train", "epoch": 58}
{"loss": 0.41895249485969543, "acc": 0.8265624642372131, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.31600096821784973, "acc": 0.8687081336975098, "length": 3.9999167919158936, "mode": "train", "epoch": 59}
{"loss": 0.4148826003074646, "acc": 0.8353515267372131, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.3205294907093048, "acc": 0.8647194504737854, "length": 3.9999935626983643, "mode": "train", "epoch": 60}
{"loss": 0.4227169454097748, "acc": 0.8285155892372131, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7724999189376831
